environment.yml

```diff
@@ -1,4 +1,6 @@
 channels:
-    - conda-forge
```

Comment:
These changes would appear to be a duplicate of those in #54 - I guess this is why you're concerned about the merge issues?